AFQ/tests/test_api.py

```diff
@@ -161,12 +161,44 @@ def test_AFQ_data():
             segmentation='freesurfer',
             use_prealign=use_prealign)
         npt.assert_equal(nib.load(myafq.b0[0]).shape,
-                        nib.load(myafq['dwi_file'][0]).shape[:3])
+                         nib.load(myafq['dwi_file'][0]).shape[:3])
         npt.assert_equal(nib.load(myafq.b0[0]).shape,
-                        nib.load(myafq.dti[0]).shape[:3])
+                         nib.load(myafq.dti[0]).shape[:3])
         myafq.export_rois()
 
 
+# def test_AFQ_slr():
```

Comment:
Maybe instead of commenting these out we can mark them for skipping as slow and implement the following pattern: https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option

Reply:
I copied the code for contest.py from that website. I added a link to the website in conftest.py . Is this a good way to document where the code came from?

Reply:
Yes - that works well. Are we using this in the CI to skip the slow tests?

Reply:
Yup! The two slow tests I added are now being skipped.